plugins/woocommerce/src/Admin/API/ProductsLowInStock.php

```diff
@@ -236,7 +237,6 @@ protected function get_query( $siteside_only = false ) {
 			  AND wc_product_meta_lookup.stock_quantity IS NOT NULL
 			  AND wc_product_meta_lookup.stock_status IN('instock', 'outofstock')
 			  :postmeta_wheres
-			order by wc_product_meta_lookup.product_id DESC
 			limit %d, %d
```

Comment:
I imported the same products but am seeing the API returns the different results. I'm wondering if it may be due to this `limit` condition?

Trunk:
<img width="1352" alt="Screenshot 2023-08-01 at 15 17 33" src="https://github.com/woocommerce/woocommerce/assets/4344253/3dc64d76-9456-4f05-bb09-4653016e98b7">

This branch:
<img width="1352" alt="Screenshot 2023-08-01 at 15 17 42" src="https://github.com/woocommerce/woocommerce/assets/4344253/329ddd72-8953-4a02-8614-d2fd735fc425">

Reply:
Thanks @chihsuan! I'll look into it.